hq/test/logic/SnykDisplayTest.scala

```diff
@@ -0,0 +1,291 @@
+       |      "id": "3333333333"
+       |    }
+       |  ]
+       |}
+     """.stripMargin
+
+  private val mockGoodAndNotVulnerableResponse =
+    s"""
+       |{
+       |  "ok": true,
+       |  "issues": {
+       |    "vulnerabilities": [],
+       |    "licenses": []
+       |  },
+       |  "dependencyCount": 0,
+       |  "packageManager": "sbt"
+       |}
+     """.stripMargin
+
+  private val mockGoodButVulnerableResponse =
+    s"""
+       |{
+       |  "ok": false,
+       |  "issues": {
+       |    "vulnerabilities": [
+       |      {
+       |        "id": "4444444444",
+       |        "url": "https://snyk.io/vuln/4444444444",
+       |        "title": "The Title",
+       |        "type": "vuln",
+       |        "description": "The description",
+       |        "from": [
+       |          "from1",
+       |          "from2",
+       |          "from3"
+       |        ],
+       |        "package": "The package",
+       |        "version": "The version",
+       |        "severity": "high",
+       |        "language": "The language",
+       |        "packageManager": "the package manager",
+       |        "semver": {
+       |          "unaffected": ">=the version",
+       |          "vulnerable": "<the version"
+       |        },
+       |        "publicationTime": "2015-11-06T02:09:36.182Z",
+       |        "disclosureTime": "2015-11-03T07:15:12.900Z",
+       |        "isUpgradable": true,
+       |        "isPatchable": true,
+       |        "identifiers": {
+       |          "CVE": [],
+       |          "CWE": [],
+       |          "NSP": 57,
+       |          "ALTERNATIVE": [
+       |            "5555555555"
+       |          ]
+       |        },
+       |        "credit": [
+       |          "Mickey Mouse"
+       |        ],
+       |        "CVSSv3": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N",
+       |        "cvssScore": 7.5,
+       |        "patches": [
+       |          {
+       |            "id": "6666666666",
+       |            "urls": [
+       |              "https://example.com/6666666666.patch"
+       |            ],
+       |            "version": "<the version >=minversion",
+       |            "comments": [
+       |              "https://example.com/7777777777.patch"
+       |            ],
+       |            "modificationTime": "2015-11-17T09:29:10.000Z"
+       |          }
+       |        ],
+       |        "upgradePath": [
+       |          true,
+       |          "the first upgrade",
+       |          "the second upgrade"
+       |        ]
+       |      }
+       |    ],
+       |    "licenses": []
+       |  },
+       |  "dependencyCount": 0,
+       |  "packageManager": "the package manager"
+       |}
+     """.stripMargin
+
+  val goodOrganisation =
+    SnykOrganisation("name0", "id0")
+
+  val goodProjects = List(
+    SnykProject("name1", "id1", None),
+    SnykProject("name2", "id2", None)
+  )
+  val goodVulnerabilities = List(
```

Comment:
This is only used in one block of tests so I don't see a reason for it to be hidden away down here? Maybe stick it in the block so the tests can share it? Generally, better to have the test setup where it's used so there are fewer places to look to comprehend the tests.

Reply:
Done.  In fact, the whole lot eventually found better homes and the addition object went away.